generators/server/templates/pom.xml.ejs

```diff
@@ -84,8 +84,10 @@
         <javassist.version>3.22.0-GA</javassist.version>
         <!-- The liquibase version should match the one managed by
         https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/${spring-boot.version} -->
-        <liquibase.version>3.5.5</liquibase.version>
+        <liquibase.version>3.6.2</liquibase.version>
         <liquibase-hibernate5.version>3.6</liquibase-hibernate5.version>
+        <h2.version>1.4.197</h2.version>
```

Comment:
as only need by h2disk in dev, it could be with conditional: `<%_ if (devDatabaseType === 'h2Disk') { _%>`